app/src/shared/components/Layout/Nav.jsx

```diff
@@ -112,7 +107,7 @@ const UnstyledWide = (props) => {
             )}
             {!!currentUser && (
                 <Fragment>
-                    {!!streamId && (
+                    <ActivityList>
```

Comment:
Here, showing off a bit of extra flexibility. `ActivityList` will work w/o `children` (defaults to `<Items />`). It'll also work with custom wrapper around its `<Items />`, a dropdown, for instance! 🚀